src/utils/api-client/api-client.ts

```diff
@@ -112,23 +112,6 @@ export class AxiosApiClient {
 	}
 
 	private _handleError(error: any) {
-		if (error.code === "ECONNABORTED") {
-			return Promise.reject("timeout");
-		}
-
-		if (error.response) {
-			// The request was made and the server responded with a status code
-			// that falls out of the range of 2xx
-			return Promise.reject({ ...error.response.data, httpStatus: error.response.status });
-		}
-
-		if (error.request) {
-			// The request was made but no response was received
-			// `error.request` is an instance of XMLHttpRequest in the browser and an instance of
-			// http.ClientRequest in node.js
-			return Promise.reject(error.request);
-		}
-
 		// Something happened in setting up the request that triggered an Error
```

Comment:
since it is throwing all errors, can remove the comment as it is not meaningful anymore

Reply:
thanks, I've removed the comment